src/main/kotlin/ArchivesMenu.kt

```diff
@@ -0,0 +1,50 @@
+class ArchivesMenu {
+    val archivesMap: MutableMap <Int, Archive> = mutableMapOf()
+    var numberOfArchives: Int = 0
+
+    private fun newArchive(name: String) {
+        val newArchive: Archive = Archive(name)
+        val currentNumber: Int = numberOfArchives + 1
+        archivesMap.put(currentNumber, newArchive)
+        numberOfArchives = currentNumber
+    }
+
+    fun showMenu() {
+        while (true) {
+            println("Список архивов:")
+            var lastNumber: Int = 0
```

Comment:
Если ты сразу инициализируешь значение - тип данных можно не писать
```suggestion
            var lastNumber = 0
```